code/datums/traits/good.dm

```diff
@@ -175,8 +175,8 @@
 
 /datum/quirk/spiritual/on_process()
 	var/comforted = FALSE
-	for(var/mob/living/L in oview(5, quirk_holder))
-		if(L.mind?.isholy && L.stat == CONSCIOUS)
+	for(var/mob/living/carbon/human/H in oview(5, quirk_holder))
```

Comment:
This is cringe but out of scope of the PR.

Reply:
Quirk `on_process()` procs are called every time when `process()` happens upon quirk, so I naturally included that stuff in the review, but I don't think this is *that* important now. What I consider more urgent(?) is qdel failures of quirk datum, which might or might not be harmful at all in the first place.